content/en/contribute/_index.md

```diff
@@ -48,5 +48,19 @@ If you would like to be notified about other people's comments or suggestions, y
 9. Representatives from the Open Modeling Foundation will review your changes for inclusion in the public version of the document.
 10. You can [track the status of your pull request and its proposed changes on the GitHub pull requests tab](https://github.com/openmodelingfoundation/openmodelingfoundation.github.io/pulls). You should receive email notifications when the proposed change has received additional comments or questions, been reviewed, or been merged into the main document.
 
+## How to Vote on a Pull Request
+
+1.	If you would like to vote on a pull request, you can do so using the **_thumbs up_** or **_thumbs down_** emoji.
+2.	From the list of pull requests select the specific topic you wish to vote on.
```

Comment:
For point 2, perhaps add "(Click the "filter" button to make sure you are seeing "open issues and pull requests" to see all of them.)

Reply:
I don't think we want to add that. Since we are only using pull requests to vote, allowing members to see open issues and pull requests would be confusing for the how to vote text. The default setting for the page is that all open pull requests will be visible when the link is followed.

Reply:
@seanbergin fair point - should we remove the text I added about filtering then?

Reply:
Probably - it is an unnecessary step and has the potential to confuse people...